src/main/config/config.c

```diff
@@ -660,11 +666,6 @@ static void resetConf(void)
 
 #if defined(ALIENFLIGHT) 
     featureClear(FEATURE_ONESHOT125);
-#ifdef ALIENFLIGHTF1
-    masterConfig.serialConfig.portConfigs[1].functionMask = FUNCTION_RX_SERIAL;
-#else
-    masterConfig.serialConfig.portConfigs[2].functionMask = FUNCTION_RX_SERIAL;
```

Comment:
Thanks, this was an fast fix.

What you advice was alredy on my plan. The AlienFlight and the Colibri_Race are the only target with secific setting left in the config.c file.